docs/DafnyRef/Attributes.md

```diff
@@ -669,7 +669,7 @@ Overrides the `/subsumption` command-line setting for this assertion.
 `{:subsumption 0}` checks an assertion but does not assume it after proving it.
 You can achieve the same effect using [labelled assertions](#sec-labeling-revealing-assertions).
 
-## 11.4. Attributes on variable declarations
+## 11.4. Attributes on variable declarations {#sec-assumption}
```

Comment:
This doesn't seem like the right label here - shouldn't it be on 11.4.1 below?

Reply:
Correct. Fixed.